City-of-Helsinki/drupal-asuntotuotanto
# ASU-X_lastest

Multiple pr:s in one

How to test:

- Get database from testing environment
- Make fresh
- drush uli --uid=86

ASU-1568: asumisoikeusnumeron näyttäminen & ASU-1567 hetun piilotus
When logged in as user 86:
- go to /fi/application/44
  - You should see asumisoikeusnumero readonly field
  - You should not see hetu-field
